### PR TITLE
Add more logging to tracking numbers

### PR DESCRIPTION
**Pull request by jomizrahi92**
This seemed useful to me, hopefully useful to others. When uploads were happening to multiple sites, I wasn't aware which site it was currently uploading to. This adds more logging around that.

### CODE REVIEW

File: lib/group_site_manager.py [hunk 1]
```diff
@@ -437,12 +439,14 @@ def _upload_oaks(self, numbers) -> None:
       driver.execute_script(f"document.getElementsByTagName('textarea')[0].value = '{js_input}';")
       driver.find_element_by_id('ContentPlaceHolder1_btnGrabar').click()
       time.sleep(2)
+      print("Upload complete for oaks.")
```
Comment (gbrodman) [inline lib/group_site_manager.py:442]: Why not just print the "finished" notification at the end of `_upload_to_group` ?

Comment (jomizrahi92) [reply lib/group_site_manager.py:442]: Just took a look and it looks like the function is returning before we can hit it, for example `_upload_oaks` gets returned from `_upload_to_group` so the finished notification won't get hit.

Comment (gbrodman) [reply lib/group_site_manager.py:442]: well that's fixable :)

Comment (jomizrahi92) [reply lib/group_site_manager.py]: Updated to address the returning

File: lib/group_site_manager.py [hunk 2]
```diff
@@ -385,23 +384,26 @@ def _melul_get_tracking_pos_costs_maps(self, group: str, username: str,
 
   def _upload_to_group(self, numbers: List[str], group: str) -> None:
     last_ex = None
-    print("Uploading tracking numbers to %s..." % group)
     for attempt in range(MAX_UPLOAD_ATTEMPTS):
       try:
+        print("Uploading to %s." % group)
```
[the remark this answers is not in the document]
Comment (jomizrahi92) [reply lib/group_site_manager.py:389]: Just to know which group we are actually uploading to, can remove if you think it's over kill.
In my experience now I've had uploads from 1-6 groups and not sure which group it was up to or which was done so I found more logs to be useful. But can totally remove if you think.

**Review by gbrodman — APPROVED** [review]

**Pull request merged**